native/src/components/CategoryListItem.tsx

```diff
@@ -29,9 +30,9 @@ const DirectionContainer = styled.View<DirectionContainerPropsType>`
   display: flex;
   flex-direction: ${props => contentDirection(props.language)};
 `
-const CategoryEntryContainer = styled.View`
+const CategoryEntryContainer = styled(View)<{ language: string }>`
```

Comment:
should be the same as
```suggestion
const CategoryEntryContainer = styled.View<{ language: string }>`
```